### PR TITLE
[skip ci] Rbd mirror admin key permissions

### PR DESCRIPTION
**Pull request by colonwq**
the updates the rbd-mirror role to copy ceph admin keyring with 600 permissions and the rest of the file list with 644 permissions. this code snip was borrowed from the mgr role.

### CODE REVIEW

File: roles/ceph-rbd-mirror/tasks/docker/copy_configs.yml [hunk 1]
```diff
@@ -23,7 +23,21 @@
   register: statconfig
   when: "item | length > 0"
 
-- name: try to fetch ceph config and keys
+- name: try to fetch ceph keys
+  copy:
+    src: "{{ fetch_directory }}/{{ fsid }}/{{ item.0 }}"
+    dest: "{{ item.0 }}"
+    owner: "{{ ceph_uid }}"
+    mode: 0600
```
Comment (andymcc) [inline roles/ceph-rbd-mirror/tasks/docker/copy_configs.yml]: You can do this in one task:
mode: "{{ (item.0 | search("keyring")) | ternary('0600', '0644') }}"

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-bluestore_osds_container

File: roles/ceph-rbd-mirror/tasks/docker/copy_configs.yml (hunk 1 above)
Comment (guits) [reply roles/ceph-rbd-mirror/tasks/docker/copy_configs.yml]: @colonwq could you please address @andymcc's comment ?

**Comment by leseb** [comment]
@colonwq please squash your commits.

**Comment by leseb** [comment]
Closing this due to inactivity, feel free to re-open, thanks!

**Pull request closed without merging**